core/server/helpers/t.js

```diff
@@ -0,0 +1,29 @@
+// # t helper
+// i18n: Translatable handlebars expressions for templates of the front-end and themes.
+// Front-end: .hbs templates in core/server, overridden by copies in themes. Themes: in content/themes.
+//
+// Usage examples, for example in .hbs theme templates:
+// {{t "Get the latest posts delivered right to your inbox"}}
+// {{{t "Proudly published with {ghostlink}" ghostlink="<a href=\"https://ghost.org\">Ghost</a>"}}}
+//
+// To preserve HTML, use {{{t}}}. This helper doesn't use a SafeString object which would prevent escaping,
+// because often other helpers need that (t) returns a string to be able to work as subexpression; e.g.:
+// {{tags prefix=(t " on ")}}
+
+var proxy = require('./proxy'),
+    jp = require('jsonpath'),
+    i18n = proxy.i18n;
+
+module.exports = function t(text, options) {
+    var bindings = {},
+        path, prop;
+    for (prop in options.hash) {
+        if (options.hash.hasOwnProperty(prop)) {
+            bindings[prop] = options.hash[prop];
+        }
+    }
+    bindings.defaultString = text;
+    bindings.isThemeString = true;
+    path = jp.stringify(['$', text]);
```

Comment:
What is the reason having jsonpath here?
This makes the helper independent from how the i18n helper is implemented.

Reply:
https://github.com/TryGhost/Ghost/pull/8437/files#diff-80f57cf21538397f7a6c7eeb1959b6a9R117
Should belong here, or?

Reply:
As suggested, moved jsonpath's `jp.stringify` method from _t.js_ to _i18n.js_. This allows more consistent code and a simplification of the default string handling.